buildkite/aggregate_incompatible_flags_test_result.py

```diff
@@ -34,6 +34,10 @@
 
 REPO_PATTERN = re.compile(r"https?://github.com/(?P<owner>[^/]+)/(?P<repo>[^/]+).git")
 
+EMOJI_PATTERN = re.compile(r":([^:\s]+):")
+
+EMOJI_IMAGE_TEMPLATE = '<img src="https://github.com/buildkite/emojis/blob/master/img-buildkite-64/{}.png?raw=true" height="16"/>'
```

Comment:
If we use this URL, we save an HTTP redirect:

```
https://raw.githubusercontent.com/buildkite/emojis/master/img-buildkite-64/{}.png
```

Reply:
Note that this will not be the correct URL for Unicode emojis like 🚀👨🏻‍💻🎉, only for custom Buildkite emojis, so it will not work with all that we might encounter. Maybe it's good enough though?

Reply:
Does BK support unicode emojis? As far as I can see we only use Buildkite's emojis in pipeline names.